tests/pytest/run_files/job_jenkins_sarus

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -l
+#SBATCH --job-name="FV3gfs_800km_resolution"
+#SBATCH --partition=cscsci
+#SBATCH --constraint=gpu
```

Comment:
Are you going to stick with running on the GPU partition of Daint?  Might be nice to have an option to switch to the multi-core partition.  (You would only need to change the constraint line to point to 'mc' instead of 'gpu')